Cython/Compiler/TreeFragment.py

```diff
@@ -38,7 +38,7 @@ def find_module(self, module_name, relative_to=None, pos=None, need_pxd=1, absol
         return ModuleScope(module_name, parent_module=None, context=self)
 
 
-def parse_from_strings(name, code, pxds=None, level=None, initial_pos=None,
+def parse_from_strings(name, code, fault_tolerant=False, pxds=None, level=None, initial_pos=None,
```

Comment:
Add `fault_tolerant=False` as the last parameter (otherwise it'd change the api and break callers).